src/components/pages/improve-communication-experience/painRestatement/painRestatement.jsx

```diff
@@ -6,28 +6,29 @@ import missed from './images/missed.svg';
 import negative from './images/negative.svg';
 import notifications from './images/notifications.svg';
 
-const TITLE = 'Effective Notification Infrastructure';
-const SUBTITLE = 'Users can tell when a notification system is poorly designed or managed.';
+const TITLE = 'Poor notifications experiences hurt your business';
+const SUBTITLE =
+  'End users expect seamless, on-brand, and timely notifications that add to their experience.';
 
 const CARDS = [
   {
-    title: 'Missed Updates',
+    title: 'Missed updates',
     description:
-      'Without timely notifications, users may miss critical updates or events related to their interests or activities, which may lead to frustration.',
+      'Without timely notifications, users miss critical information or updates such as order updates, password resets, or one-time password messages, leading to a poor experience with your application.',
```

Comment:
Maybe instead of:

Without timely notifications, users miss critical information or updates such as order updates, password resets, or one-time password messages, leading to a poor experience with your application

we use

Without timely notifications, users miss critical information such as order updates, password resets, or one-time password messages, leading to a poor experience with your application.

(the double "updates" bugged me)

Reply:
Yes! great suggestion.